### DISCUSSION TITLE
Designspace V5, what’s possible?

### DISCUSSION THREAD

**Discussion by jenskutilek** [fonttools/fonttools#2630] [Q&A] [answered]
I tried to upgrade some of my designspace files to v5, and it’s great what’s now possible from one file where previously a lot of different parts were involved. Congrats to everyone making it possible!

I wonder if it's possible to build both a Roman and Italic VF from the same designspace file, where both VFs are not outline-compatible. I tried but failed. It feels like it might be possible, as 99.9% of the data is identical for both the Roman and the Italic separate designspace files. Am I missing something?

**Comment by belluzj** [comment, marked as answer]
It should work out of the box with [fontmake](https://github.com/googlefonts/fontmake/releases/tag/v3.4.0a2) and [ufo2ft pre-releases](https://github.com/googlefonts/ufo2ft/releases/tag/v2.28.0a1) if you define a discrete axis for your Italic axis, and 2 `<variable-font>` elements. You could take inspiration from these test files:

https://github.com/fonttools/fonttools/blob/42e4d66184be1acda05bed1896da6d61c084835e/Tests/designspaceLib/data/test_v5_sourceserif.designspace#L30-L35

https://github.com/fonttools/fonttools/blob/42e4d66184be1acda05bed1896da6d61c084835e/Tests/designspaceLib/data/test_v5_sourceserif.designspace#L168-L221

https://github.com/fonttools/fonttools/blob/42e4d66184be1acda05bed1896da6d61c084835e/Tests/designspaceLib/data/test_v5_MutatorSans_and_Serif.designspace#L4-L9

https://github.com/fonttools/fonttools/blob/42e4d66184be1acda05bed1896da6d61c084835e/Tests/designspaceLib/data/test_v5_MutatorSans_and_Serif.designspace#L120-L143

**Reply by jenskutilek** [reply]
Thank you! I was missing the fontmake prerelease version :)

**Reply by belluzj** [reply]
Great! Please comment about your experience with the pre-release so we can fix bugs if any and make it a real release soon :)
